### DISCUSSION TITLE
Minimize cannonball icon

### DISCUSSION THREAD

**Discussion by pettrs** [runelite/runelite#14937] [Ideas]
The cannonball icon is way too big on fixed client mode. possible remove when you're within x tiles, minimize it by default, or add option to remove overall

![image](https://user-images.githubusercontent.com/75093127/167879159-c587fc3f-48a6-4d26-85d5-873b2c986a53.png)

**Comment by Enriath** [comment]
This overlay is just for showing the "good cannon spots". It can be separately disabled in the plugin's settings, and only shows when you have a full cannon in your inventory. As such, it shouldn't show up very often overall. If you've got common spots that you use a lot, you could mark them with `Ground Markers` instead.

**Reply by pettrs** [reply]
I would like to use it in it's current state, i find it very useful. is it not possible to just minimize the icon by like 50% so you can at least see what's under it?
